### ISSUE TITLE
standardImplementation script in html

### ISSUE THREAD

**Issue by rockneverdies55** [react-ga/react-ga#333]
[In README](https://github.com/react-ga/react-ga#standard-implementation), the script to add google analytics manually has `<%=htmlWebpackPlugin.options.analyticsURL%>` in it. Is it intentional to have this in there instead of regular url of ga.js? I don't have html webpack plugin installed at all in my project? Should I maybe replace that with `https://www.google-analytics.com/analytics.js` as stated in https://developers.google.com/analytics/devguides/collection/analyticsjs/ page?

**Comment by SimeonC** [comment]
Yes, you are correct - it should be what you have in the analytics page
